2-mandatory/2-financial-times.js

```diff
@@ -14,24 +20,53 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    let shortestHeadline;
+    let fewestNumberOfWords;
+    for(let title of allArticleTitles){
+        let numberOfWords = title.split(' ').length;
+        if (fewestNumberOfWords === undefined || numberOfWords < fewestNumberOfWords){
```

Comment:
good job! you have used || in if function but I have used greater > in if function.